src/platform/web/Platform.js

```diff
@@ -307,6 +315,48 @@ export class Platform {
         return DEFINE_VERSION;
     }
 
+    get themes() {
```

Comment:
perhaps you could put all of the theme related code into a separate class that is accessible on `platform.theming` or something to prevent Platform from growing too big?